apps/storybook/src/Surface.stories.tsx

```diff
@@ -39,3 +50,180 @@ export const Basic: Story<SurfaceProps> = ({ elevation }) => {
 };
 
 Basic.args = { elevation: 1 };
+
+export const Custom: Story<SurfaceProps> = (args) => {
```

Comment:
I made a story for each of the cases shown in the surface demo pages. Let me know if any of them can be removed. I'm also open to suggestions about these story names